rt_dashboard/templates/rt_dashboard/dashboard.js

```diff
@@ -11,6 +11,22 @@ var url_for_jobs = function(param, page) {
     return url;
 };
 
+function getCookie(name) {
+    let cookieValue = null;
+    if (document.cookie && document.cookie !== '') {
+        const cookies = document.cookie.split(';');
+        for (let i = 0; i < cookies.length; i++) {
+            const cookie = cookies[i].trim();
+            // Does this cookie string begin with the name we want?
+            if (cookie.substring(0, name.length + 1) === (name + '=')) {
+                cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
```

Comment:
another way of doing it is
```suggestion
            const [cookieName, cookieValue] = cookies[i].trim().split('=', 2);
            if (cookieName === name) {
                cookieValue = decodeURIComponent(cookieValue);
```